contracts/actions/summerfi/SFApproveTokens.sol

```diff
@@ -0,0 +1,128 @@
+// SPDX-License-Identifier: MIT
+pragma solidity =0.8.10;
+
+import {ActionBase} from '../ActionBase.sol';
+import {IERC20} from '../../interfaces/IERC20.sol';
+import {IDSProxy} from '../../interfaces/IDSProxy.sol';
+import {IExecutable} from '../../interfaces/summerfi/IExecutable.sol';
+import {IOperationsRegistry} from '../../interfaces/summerfi/IOperationsRegistry.sol';
+import {IOperationExecutor, Call} from '../../interfaces/summerfi/IOperationExecutor.sol';
+
+
+/// @title Approve tokens through Summerfi proxy
+contract SFApproveTokens is ActionBase {
+    address constant SF_OPERATION_EXECUTOR = 0xcA71C36D26f515AD0cce1D806B231CBC1185CdfC;
+    address constant SF_OPERATIONS_REGISTRY = 0x563d2689fE89c78259dD7F694146BB93f6388A55;
+    string constant SF_OPERATION_NAME = 'AAVEV3PaybackWithdraw';
```

Comment:
We usually use double quotes in contracts, same goes for imports 😄

Reply:
auto format, my bad :(